native/src/core/su/su.cpp

```diff
@@ -38,23 +38,20 @@ int quit_signals[] = { SIGALRM, SIGABRT, SIGHUP, SIGPIPE, SIGQUIT, SIGTERM, SIGI
     "MagiskSU\n\n"
     "Usage: su [options] [-] [user [argument...]]\n\n"
     "Options:\n"
-    "  -c, --command COMMAND         Pass COMMAND to the invoked shell\n"
-    "  -i, --interactive             Force pseudo-terminal allocation when using -c\n"
+    "  -s, --shell SHELL             Use SHELL instead of the default " DEFAULT_SHELL "\n"
+    "  -i, --interactive             Force pseudo-terminal allocation\n"
```

Comment:
since there's a breaking change on `-m`, how about making tty allocation `-t` as `ssh`, `docker`, and changing attach mount namespace to `-m`?

Reply:
No options have changed

Reply:
`--mount-master` vs `-t 0`?

Reply:
They are the same, I removed the obsolete -mm help message.